Lethal-Lemurs/nodejs-bangazon-site
# Db get orders

## Status
**READY**

## Description
The user can now select to see and individual product categories or all the product categories with a drop down in the navbar that is shown on every single page

## Resolves Issue Number
#15 and work towards tickets #11 and #4 

## TEST
- drop migrations
- fetch branch to test
- migrate and seed
- login and navigate to product details and add a product to an order. You should then see all of the products on that order. Add a different product to the order and it should appear as well.


## Impacted Files in Application
    modified:   controllers/ProductDetailsCrtl.js
    modified:   routes/ProductDetailsRoute.js
